<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DokoPlayerStatsSpecialStats</class>
 <widget class="QWidget" name="DokoPlayerStatsSpecialStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelHochzeitenHeader">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Hochzeiten:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHochzeiten">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelHochzeitenWonHeader">
         <property name="text">
          <string>Won:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHochzeitenWon">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelAbgabenHeader">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Abgaben:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAbgaben">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelAbgabenWonHeader">
         <property name="text">
          <string>Won:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAbgabenWon">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelSchweinereienHeader">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Schweinereien:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSchweinereien">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelSchweinereienWonHeader">
         <property name="text">
          <string>Won:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSchweinereienWon">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelSoliHeader">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Soli:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSoli">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelSoloPflichtHeader">
         <property name="text">
          <string>Pflicht:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSoloPflicht">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
